api/views/__init__.py

```diff
@@ -184,6 +190,21 @@ def runtime_data(request):
             "WAFFLE_FLAGS": flag_values,
             "WAFFLE_SWITCHES": switch_values,
             "WAFFLE_SAMPLES": sample_values,
-            "MAX_MINUTES_TO_VERIFY_REAL_PHONE": MAX_MINUTES_TO_VERIFY_REAL_PHONE
+            "MAX_MINUTES_TO_VERIFY_REAL_PHONE": MAX_MINUTES_TO_VERIFY_REAL_PHONE,
         }
     )
+
+
+@swagger_auto_schema(methods=["post"], request_body=WebcompatIssueSerializer)
+@decorators.api_view(["POST"])
+@decorators.permission_classes([permissions.IsAuthenticated])
+def report_webcompat_issue(request):
```

Comment:
suggestion (non-blocking): add a python doc string here for the API docs. In particular, when I tried to send `issue_on_domain: "disney.com"`, I got an error. Would be good to include a doc describing more detail about each field.